### PR TITLE
[Feature][Vectorized] Support explode json array func #8526

### PR DESCRIPTION
**Pull request by Lchangliang**
# Proposed changes

Issue Number: close #xxx

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: be/src/vec/functions/function_fake.h [hunk 1]
```diff
@@ -40,6 +40,26 @@ struct FunctionExplodeSplitImpl {
     }
 };
 
+struct FunctionExplodeJsonArrayIntImpl {
+    static constexpr auto name = "explode_json_array_int";
+    static DataTypePtr get_return_type_impl(const DataTypes& arguments) {
+        return std::make_shared<DataTypeUInt8>();
```
Comment (BiteTheDDDDt) [inline be/src/vec/functions/function_fake.h:46]: Why return type is UINT8 here?

**Comment by BiteTheDDDDt** [comment]
LGTM

**Comment by Lchangliang** [comment]
> Hi @Lchangliang , please rebase to resolve the conflict

I already rebase the code and check it.

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**